src/css/layout/helpers.less

```diff
@@ -35,3 +35,7 @@
 .base-bottom {
   padding-bottom: @base-spacing-unit;
 }
+
+.inline-block {
+  display: inline-block;
```

Comment:
I'd rather have `style={{"display": "inline-block"}}` directly on the component. Alternatively we could think of changing the class name to something meaningful ...